Mathlib/NumberTheory/LegendreSymbol/AddCharacter.lean

```diff
@@ -245,13 +254,13 @@ noncomputable def primitiveCharFiniteField (F F' : Type*) [Field F] [Finite F] [
     rw [one_mul] at ha
     exact ⟨a, fun hf => ha <| (ψ.prim.zmod_char_eq_one_iff pp <| Algebra.trace (ZMod p) F a).mp hf⟩
   exact ⟨ψ.n, ψ', hψ'.isPrimitive⟩
-#align add_char.primitive_char_finite_field AddChar.primitiveCharFiniteField
+#align add_char.primitive_char_finite_field AddChar.FiniteField.primitiveChar
+@[deprecated (since := "2024-05-30")] alias primitiveCharFiniteField := FiniteField.primitiveChar
```

Comment:
EDIT: It seems this isn't actually true, so please ignore this remark. (I think it may have been true with a previous version of the deprecation syntax, or maybe I just hallucinated it completely.)

Reply:
This is the syntax that you get when you type "de" in VSCode and accept the first suggestion...